src/components/button/component.js

```diff
@@ -429,6 +430,26 @@ export let Button = xcomponent.create({
                 if (style.label === 'buynow' && style.size === 'tiny') {
                     throw new Error(`Invalid ${style.label} button size: ${style.size}`);
                 }
+
+                if (style.label === 'buynow' && style.fundingicons !== undefined && style.branded !== undefined) {
+                    throw new Error(`Must use either of fundingicons or branded option for ${style.label} button`);
+                }
+
+                if (style.label === 'buynow' && style.branded === false) {
```

Comment:
shouldnt we throw error if the user is trying to use branding option with anything other than buynow?